qcengine/programs/openmm.py

```diff
@@ -0,0 +1,182 @@
+"""
+Calls the OpenMM executable.
+
+Requires RDKit
+"""
+import json
+import os
+from typing import Dict
+
+from qcelemental.models import Result
+
+from .model import ProgramHarness
+from ..exceptions import InputError, RandomError, ResourceError, UnknownError
+from ..util import execute, popen, temporary_directory
+from ..units import ureg
+
+
+class OpenMMHarness(ProgramHarness):
+
+    # TODO: verify correctness of these default params
+    # where are these used? By QCFractal?
+    _defaults = {
+        "name": "OpenMM",
+        "scratch": True,
+        "thread_safe": False,
+        "thread_parallel": True,
+        "node_parallel": False,
+        "managed_memory": True,
+    }
+
+    class Config(ProgramHarness.Config):
+        pass
+
+    @staticmethod
+    def found(raise_error: bool = False) -> bool:
+        # TODO: this harness requires RDKit as well, so this needs checking too
+        return which_import('simtk.openmm',
+                     return_bool=True,
+                     raise_error=raise_error,
+                     raise_msg='Please install via `conda install openmm -c omnia`.')
+
+    @staticmethod
+    def _process_molecule_rdkit(jmol):
+        from rdkit import Chem
+        from rdkit.Chem import AllChem
+
+        # Handle errors
+        if abs(jmol.molecular_charge) > 1.e-6:
+            raise InputError("RDKit does not currently support charged molecules.")
```

Comment:
@dgasmith I don't have a strong opinion on where this goes; your view is more global than mine. It might make sense to put it as a standalone utility function in the `rdkit.py` program module and then import it here. We already import the `RDKitHarness` here since we call its `found` method, so this wouldn't be crossing streams any more than we already are.

Reply:
Can we add this static method to the RDKit Harness then?